hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/lock/LockManager.java

```diff
@@ -57,12 +56,9 @@ public LockManager(final Configuration conf) {
    * @param fair - true to use fair lock ordering, else non-fair lock ordering.
    */
   public LockManager(final Configuration conf, boolean fair) {
-    final int maxPoolSize = conf.getInt(
-        HddsConfigKeys.HDDS_LOCK_MAX_CONCURRENCY,
-        HddsConfigKeys.HDDS_LOCK_MAX_CONCURRENCY_DEFAULT);
     lockPool =
         new GenericObjectPool<>(new PooledLockFactory(fair));
-    lockPool.setMaxTotal(maxPoolSize);
+    lockPool.setMaxTotal(-1);
```

Comment:
NIT: we don't need to explicitly set -1 as the default maxTotal from GenericObjectPool is -1.

Reply:
yes, I did this expicility, if in future released they changed the defaults it might break.

Reply:
Sounds good. +1 pending CI.